js_modules/dagster-ui/packages/ui-core/src/overview/OverviewSensorsRoot.tsx

```diff
@@ -81,16 +120,23 @@ export const OverviewSensorsRoot = () => {
   }, [data, visibleRepos]);
 
   const {state: runningState} = runningStateFilter;
+
   const filteredBuckets = useMemo(() => {
```

Comment:
This is all client-side filtering?  

Out of the scope of this PR, but wondering if it would help to support query-based filtering for orgs that have a lot of sensor definitions.

Reply:
Yeah agreed. At some point we'll need to migrate most of our querying/filtering to be server side, that will have product implications though that could require some design reworks / feature reworks.